lib/features/booking/view_model/cubit/add_inspection_booking_cubit.dart

```diff
@@ -90,6 +90,19 @@ class AddInspectionBookingCubit extends Cubit<AddInspectionBookingState> {
     }
   }
 
+  void updateProfilePhoneNumber(String phoneNo,
+      {PhoneCode countryCode = PhoneCode.au}) async {
+    try {
+      UserDetail user = await authenticationService.getCurrentUser();
+      final String formatNumber = phoneNo.substring(1); // remove first zero
```

Comment:
what if user does not enter the first number as 0?

Reply:
Actually, validation has been implemented on UI part. Phone number should match with area code ( always have zero atm)